### PR TITLE
graffiti wall tag - Michael

### PR DESCRIPTION
**Pull request by MikecraftA**
added michael.json to tags

### CODE REVIEW

**Comment by asg017** [comment]
Hey @MikecraftA!

First off - nice upvote! 🔼 

Secondly, it looks like you accidentally made your pull request to the wrong repository - which is okay, we can fix it! This can be a little confusing, so I'll explain:

This repository, `eceusc/graffiti-wall`, was the original grafitti wall project created by [ECE USC](http://eceusc.ucsd.edu). However, we (the new Open Source club at UCSD) wanted to have the same project, so we "forked" (duplicated) our own graffiti wall repository that lives at `os-ucsd/graffiti-wall`. So, for the Open Source Summer Weekly Challenge #1, we'd like to see your pull request on that repository instead of ECE USC's, to keep things consistent!

Fortunately, this is an easy fix! You can close this pull request with the `Close` button on this page. Then, go to your forked version of the graffiti wall ([link here](https://github.com/MikecraftA/graffiti-wall)!), click on the `New Pull Request` button, and the page should show something that has `base repository: os-ucsd/graffiti-wall` and `head repository: MikecraftA:master`. If you see that, then that's great, just click on the green `Create pull request` button and continue on like normal!


So, to recap:
- You forked from the correct repository! You can verify this by seeing your [forked repo](https://github.com/MikecraftA/graffiti-wall), it says `"forked from os-ucsd/graffiti-wall"` - which is what we want 
- Your tag is looking great, [the commit you made](https://github.com/MikecraftA/graffiti-wall/commit/3e0645cdbdb37d185de064cc95cd2163ed05b27d) is perfect!
- However, the "Pull Request" you made was towards the wrong repo - it should've been to the `os-ucsd/graffiti-wall` repo, but it was made to the older `eceucsd/graffiti-wall` repo. So just close the wrong PR (this one), create a new one (see the instructions above), and you're good!

I'm sorry for the confusion, I hope this explanation helps - if you have any questions, feel free to comment below!
